interface/forms/newpatient/common.php

```diff
@@ -62,6 +62,7 @@ function sensitivity_compare($a, $b) {
 <?php html_header_show();?>
 <title><?php echo xlt('Patient Encounter'); ?></title>
 
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'];?>/bootstrap-3-3-4/dist/css/bootstrap.css" type="text/css">
```

Comment:
also, when bringing in this "light light bootstrap", should we also bring in the rtl version, if pertinent?